### PR TITLE
[1.0] Update LP and Broker API docs with RpcAsset type

### PR DESCRIPTION
**Pull request by j4m1ef0rd**
Documentation for the changes to the API's in [#4153](https://github.com/chainflip-io/chainflip-backend/pull/4153). 

Did a lot of cleanup to try and make the pages more consistent. 

Should I update the usage of `Asset` in the `Javascript SDK` sections? Not sure how to document it. @zoheb391

### CODE REVIEW

**Comment by dandanlen** [comment]
FYI. I rebased with main and updated according to the latest state of [#4153](https://github.com/chainflip-io/chainflip-backend/pull/4153).

File: pages/integration/liquidity-provision/lp-api.mdx [hunk 1]
```diff
@@ -143,11 +155,11 @@ Return:
 
 - Egress id, which is used as the identifier for the operation/s to egress/withdraw the requested funds.
 
-Example:
+#### Example
 
 - Request
   ```sh
-  curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "lp_withdraw_asset", "params": [1000, "Usdc", "0x71C7656EC7ab88b098defB751B7401B5f6d8976F"]}' http://localhost:10589
+  curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "lp_withdraw_asset", "params": [1000, "USDC", "Ethereum", "0x71C7656EC7ab88b098defB751B7401B5f6d8976F"]}' http://localhost:10589
```
Comment (martin-chainflip) [inline pages/integration/liquidity-provision/lp-api.mdx:162]: Is this change correct? According to the documentation above, this method takes three arguments. I would either expect `"ETH"` (i.e. the simplified, unambiguous currency) or `{chain:"Ethereum", asset:"ETH"}` as the second argument

File: pages/integration/liquidity-provision/lp-api.mdx [hunk 2]
```diff
@@ -189,15 +210,24 @@ Parameters:
 Return:
 
 - A chronological list of all the updates that were done as part of the operation. Each item in the list is a JSON object:
-  `{ "tick_range": [<tick>, <tick>], "liquidity_total": <liquidity>, "collected_fees": { "base": <amount> , "pair": <amount> }, "increase_or_decrease": <increase_or_decrease>, "liquidity_delta": <liquidity>, "assets_delta": { "base": <amount> , "pair": <amount> } }`
-  For example if a lp_setRangeOrder request changes the total size of an existing range order, the return would contain two of these, one for the decrease to zero/destruction of the previous order, and then one for the increase/creation of the order with the new size.
+```json
```
Comment (martin-chainflip) [inline pages/integration/liquidity-provision/lp-api.mdx:213]: This is not correct. The result contains
"id" and "size_change", but no longer "liquidity_delta".
also, "base_asset", "pair_asset" (both of which are not reported in the verbose form! )

Comment (dandanlen) [reply pages/integration/liquidity-provision/lp-api.mdx:213]: Updated this and the other one (and a bunch of other stuff).

**Review by martin-chainflip — APPROVED** [review]

**Pull request merged**